src/components/features/ctf-components/ctf-teaser/ctf-teaser.tsx

```diff
@@ -0,0 +1,64 @@
+import { sectionTypes } from '@src/components/shared/global';
+
+export const CtfTeaser = (
+  props: any
+) => {
+  const { title, subtitle, buttonText, image , buttonUrl, sectionType} = props;
+  if (sectionType == sectionTypes.solutionSection) {
+    return (
+      <div className="flex w-full max-w-[455px] flex-row gap-x-[75px] h-full">
+        <img
+          width="75px"
+          className="place-self-start"
+          alt="icon"
+          src={image?.url as string}
```

Comment:
Is het niet beter om de image tag in z'n geheel weg te laten als je geen image zou hebben? Anders gaat die waarschijnlijk een broken image laten zien en dan toon je denk ik beter niets...